### DISCUSSION TITLE
[Question] Any plans to support embedding on other platforms?

### DISCUSSION THREAD

**Discussion by matthew-dean** [servo/servo#22757] [General] [answered]
For work, I'm currently evaluating embedded browser engines for applications on Xbox / PS4, and possibly for Smart TV (WebOS / Tizen). Is it possible to build / embed Servo for those platforms? If not, is that something that may happen eventually?

**Comment by jdm** [comment]
Those are not platforms that we have attempted to port Servo to before. It may well be possible, but I don't know enough about them to meaningfully comment.

**Comment by fabricedesre** [comment]
My experience with getting Servo working on gonk based devices is that Servo itself is quite easy to port: you basically need to provide it a GL context and input events. The hard part can be managing to compile SpiderMonkey for your platform, because of the build system used by SM is not really flexible.

**Comment by jdm** [comment, marked as answer]
In case it's still relevant, Servo is now embeddable in UWP applications, which means it should be possible to run it on Xbox.
